book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ ] A method for adding rentals should be present.
- [ ] @rentals should be initialized. 
```ruby
    @rentals = []
```